styles/style.css

```diff
@@ -0,0 +1,237 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600&display=swap');
```

Comment:
ogólny comment - sprawdziłam wersje responsywne - ogólnie są git, trochę nie leżało mi 540x720 (surface duo) i 1024x1366 (ipad pro), ale możliwe że tutaj nic lepszego nie wykombinujemy...

Reply:
zgadzam się, wyglądają średnio, bo zdjęcie jest za małe w stosunku do opisów, ale na ten moment i przy tej konfiguracji i układzie nie jestem już w stanie nic z tym zrobić, chyba, że masz jakiś pomysł jak to zmodyfikować?